jax/interpreters/batching.py

```diff
@@ -174,6 +174,12 @@ def defbroadcasting(prim):
   primitive_batchers[prim] = partial(broadcast_batcher, prim)
 
 def broadcast_batcher(prim, args, dims, **params):
+  """Process a primitive with built-in broadcasting.
+
+  Args:
+    args: the arguments
+    dims: for each argument, the dimension that is being batched (or None)
```

Comment:
How about "dims: list or tuple of integers of the same length as `args`, where each entry is either an int indicating the batch dimension of the corresponding entry of `args` or else `not_mapped` indicating the corresponding entry of `args` is not batched"?